drivers/amazon/provider.go

```diff
@@ -78,5 +79,8 @@ func New(opts ...Option) autoscaler.Provider {
 	if p.userdata == nil {
 		p.userdata = userdata.T
 	}
+	if p.imdsTokens == "" {
+		p.imdsTokens = "optional"
+	}
```

Comment:
I recommend removing this change, per above comments about not automatically setting a value (which otherwise needs to be fully tested for regressions through setting up an environment, etc)